### DISCUSSION TITLE
Why version 5.0.0 beta-2 Windows zip package contains more than just the exe file?

### DISCUSSION THREAD

**Discussion by ShaiAvr** [premake/premake-core#1982] [Q&A] [answered]
I am getting started with `premake`, and I found something interesting that made me curious:

In previous versions of `premake` up to version 5.0.0 beta-1, the zip package for windows only included `premake5.exe` and that's it. However, in the new 5.0.0 beta-2 release, the zip package seems to have more files such as `example.dll`, `example.lib`, `premake5.pdb`, and more.
Here's a comparison of the contents of the beta-2 zip (on the right) vs the beta-1 zip (on the left). The beta-2 zip was obtained from `https://github.com/premake/premake-core/releases/download/v5.0.0-beta2/premake-5.0.0-beta2-windows.zip` and the beta-1 zip from `https://github.com/premake/premake-core/releases/download/v5.0.0-beta1/premake-5.0.0-beta1-windows.zip`.

![Screenshot 2022-10-13 010901](https://user-images.githubusercontent.com/35605235/195457196-cb154f5a-5451-4a37-91c9-6a81431c96d7.png)

What is the purpose of all the new files in the beta-2 package? Are they mandatory?
From their names, they look like debug files or something like that, but the two `exe` files weigh around the same size, which makes me believe they're both release builds (or both debug builds, which I really hope isn't the case).
Of course, I can keep them and not care about them, but I am curious as to why they are here and what I can do with them.
Thanks in advance.

**Comment by nickclark2016** [comment, marked as answer]
It's because I'm terrible at making releases artifacts. You only need the premake executable, all the rest is just other artifacts of the build you won't need.
